src/test/playwright/support/requests/ExerciseAPIRequests.ts

```
@@ -229,6 +229,28 @@ export class ExerciseAPIRequests {
         return response.json();
     }
 
+    /**
+     * Adds a text exercise to an exercise group in an exam or to a course.
+     *
+     * @param body - An object containing either the course or exercise group the exercise will be added to.
+     * @param title - The title for the text exercise (optional, default: auto-generated).
+     */
+    async createTextExerciseWithDates(body: { course: Course } | { exerciseGroup: ExerciseGroup }, title = 'Text ' + generateUUID()): Promise<TextExercise> {
```

Biome

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)